python/nav/web/api/v1/alert_serializers.py

```diff
@@ -136,7 +136,8 @@ def get_device_groups(obj):
         """Returns all the device groups for the netbox if any"""
         try:
             netbox = obj.netbox
-            return netbox.groups.values_list('id', flat=True) or None
+            values = netbox.groups.values_list('id', flat=True)
+            return list(values) if values else None
```

Comment:
You might want to check if suffixing `.exists()` to `values` work, as that is a cheaper computation on postgres, handy if `values` have many members.

Then again, that would be pre-optimizing  :)

Reply:
Looking at the available data, I'd say the most common case is that the list is empty, in fact.